tools/generate_sysroot.py

```diff
@@ -228,7 +237,13 @@ def main():
     outpath = Path(__file__).parent.parent / 'sysroot'
   outpath.mkdir(exist_ok=True)
 
-  for arch in ['arm', 'arm64', 'x86']:
+  # TODO(jsuya): x86 package does not exist in Tizen 10.0.(2025-05-27)
+  if args.api_version >= 10.0:
+    arches = ['arm', 'arm64']
```

Comment:
This also looks like temporary code.
How do you think about updating after the development has stabilized?